src/dev/pages/stack/stack.ejs

```diff
@@ -0,0 +1,119 @@
+<p class="forge-typography--body2">Resize browser window to see wrapping (set inline and wrap to true)</p>
+<div class="stack-container">
+  <forge-stack id="main-demo" gap="16px">
+    <div class="box"></div>
+    <div class="box"></div>
+    <div class="box"></div>
+    <div class="box"></div>
+  </forge-stack>
+</div>
+
+<div class="more-examples">
+  <p class="forge-typography--title">More examples</p>
+  <forge-stack gap="56">
+    <forge-card outlined>
+      <forge-toolbar style="--forge-toolbar-padding: 0" no-border>
+        <p slot="start" class="forge-typography--body2">Forge toolbar with perfectly spaced end slot buttons</p>
+      </forge-toolbar>
+      <forge-toolbar>
+        <forge-stack inline gap="16px" slot="end">
+          <forge-button>
+            <button type="button">Cancel</button>
+          </forge-button>
+          <forge-button type="unelevated">
+            <button type="button">Save</button>
+          </forge-button>
+        </forge-stack>
+      </forge-toolbar>
+    </forge-card>
+    <forge-card outlined style="--forge-card-padding: 0;">
+      <forge-toolbar>
+        <p slot="start" class="forge-typography--body2">Simple vertical form</p>
+      </forge-toolbar>
+      <form style="padding: 16px;">
+        <forge-stack gap="16">
+          <forge-text-field>
+            <input type="text" id="input-text-01" />
+            <label for="input-text-01" slot="label">Text field</label>
+          </forge-text-field>
+          <forge-text-field>
+            <input type="text" id="input-text-01" />
+            <label for="input-text-01" slot="label">Text field</label>
+          </forge-text-field>
+          <forge-text-field>
+            <input type="text" id="input-text-01" />
+            <label for="input-text-01" slot="label">Text field</label>
+          </forge-text-field>
+        </forge-stack>
+      </form>
+    </forge-card>
+
+
+    <forge-card outlined style="--forge-card-padding: 0;">
```

Comment:
I'm worried that using this as-is for complex layouts will discourage responsive design — each row in this example will keep its number of columns even in a really small container. Ideally the items should be allowed to wrap to a new line when the stack narrows.

I think this should introduce something like a `data-stack-break` attribute to tell the component where to start new lines as the container size changes. Basically like a `grid-column-start: 1` style declaration. It might look like this:
```
<forge-stack inline wrap stretch>
  <div>Full width</div>
  <div data-stack-break>Half width</div>
  <div>Half width</div>
  <div data-stack-break>Third width</div>
  <div>Third width</div>
  <div>Third width</div>
</forge-stack>
```
As the container shrinks it would gradually move to a single column layout, made possible because the inner stacks have been flattened into the base stack.

Reply:
This is another change that might require `grid`

Reply:
Being able to control where/when it wraps is a good point here that may make a better case for `grid` instead. Right now everything will collapse down into a single column, but `flex` doesn't provide an easy way like `grid` does.

I think it worth noting that we should also abstract any concept of `flex` or `grid` from the API so we can make modifying the internals easier without technologies-specific naming.